apps_otelcol.tf

```diff
@@ -186,6 +186,12 @@ resource "kubernetes_daemonset" "otel_agent" {
             }
           }
         }
+
+        toleration {
```

Comment:
Too specific. This is much too much LNCD-dependent, we need to pass `toleration`s blocks  as a variable to the module.

Reply:
I thought this is LNCD-specific repository, if you want I'll adjust it

Reply:
It's not _that_ specific - see external projects. :)